office_space_allocation/livingspace.py

```diff
@@ -0,0 +1,32 @@
+from office_space_allocation.fellow import Fellow
+from office_space_allocation.room import Room
+from office_space_allocation.staff import Staff
+from office_space_allocation.utilities import InvalidRoomOccupantError, RoomFullError
+
+
+class LivingSpace(Room):
+    def can_accept_occupants(self):
+        return self.get_num_occupants() < 4
+
+    def add_person(self, person):
+        """
+        Adds ```Person``` person to the list of occupants of the LivingRoom
+        Person should only be of type Fellow
+
+        :param : ```Person``` person
+        """
+        if self.can_accept_occupants():
+            if isinstance(person, Fellow):
+                self.occupants.append(person)
+            elif isinstance(person, Staff):
+                raise InvalidRoomOccupantError("Staff cannot join LivingRooms")
+        else:
+            raise RoomFullError("LivingSpace is full")
+
+    def __eq__(self, other_room):
```

Comment:
@kn9ts @Bopchy I have use the Room comparisons in the `find_room` function within amity.py file. I also use it within the test_database.py file when making comparison if the saved rooms are loaded correctly.